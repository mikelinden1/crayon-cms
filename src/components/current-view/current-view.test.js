import CurrentView from './current-view';

it('CurrentView matches snapshot', () => {
    const wrapper = shallow(<CurrentView items={[{}, {}]} filteredItems={[{}]} />);
    expect(wrapper).toMatchSnapshot();
});

it('CurrentView shows AddNew Jumbotron if no items', () => {
    const wrapper = shallow(<CurrentView items={[]} filteredItems={[]} />);
    expect(wrapper).toMatchSnapshot();
});

it('CurrentView shows NoMatchingItems Jumbotron if no filtered items', () => {
    const wrapper = shallow(<CurrentView items={[{}, {}]} filteredItems={[]} />);
    expect(wrapper).toMatchSnapshot();
});

it('CurrentView shows table view', () => {
    const wrapper = shallow(<CurrentView items={[{}, {}]} filteredItems={[{}]} currentView="table" />);
    expect(wrapper).toMatchSnapshot();
});

it('CurrentView shows grid view', () => {
    const wrapper = shallow(<CurrentView items={[{}, {}]} filteredItems={[{}]} currentView="grid" />);
    expect(wrapper).toMatchSnapshot();
});