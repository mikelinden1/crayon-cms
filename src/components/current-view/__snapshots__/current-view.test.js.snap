// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentView matches snapshot 1`] = `<Connect(TableView) />`;

exports[`CurrentView shows AddNew Jumbotron if no items 1`] = `
<Connect(AddNew)
  displayAsJumbotron={true}
/>
`;

exports[`CurrentView shows NoMatchingItems Jumbotron if no filtered items 1`] = `<Connect(NoMatchingItems) />`;

exports[`CurrentView shows grid view 1`] = `<Connect(GridView) />`;

exports[`CurrentView shows table view 1`] = `<Connect(TableView) />`;
